ros/src/computing/planning/motion/packages/waypoint_follower/include/mpc_follower/lowpass_filter.h

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2018-2019 Autoware Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file lowpass_filter.h
+ * @brief vehicle model interface class
+ * @author Takamasa Horibe
+ * @date 2019.05.01
+ */
+
+#pragma once
+#include <cmath>
+#include <vector>
+#include <algorithm>
+
```

Comment:
@TakaHoribe please add
```
#include <stdio.h>
```
to fix the compilation failure due to `printf` in Melodic

Reply:
I fixed it, thanks.